benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1426,6 +1434,15 @@ ppx::grfx::Format GraphicsBenchmarkApp::RenderFormat()
     return renderFormat;
 }
 
+void GraphicsBenchmarkApp::CreateColorsForDrawCalls()
+{
+    // Create colors randomly.
+    mColorsForDrawCalls.resize(kMaxSphereInstanceCount);
+    for (size_t i = 0; i < kMaxSphereInstanceCount; i++) {
+        mColorsForDrawCalls[i] = float4(mRandom.Float(), mRandom.Float(), mRandom.Float(), 0.5f);
```

Comment:
Should this random number generation be connected with the random seed `kSeed` somehow?

Reply:
The default seed is also fixed in BigWheels, so fine with both.